deps.sbt

```diff
@@ -1,6 +1,6 @@
 libraryDependencies ++= Seq(
-  "ohnosequences" %% "s3"      % "0.1.0-4-gaa153b0",
-  "ohnosequences" %% "forests" % "0.1.0"
+  "ohnosequences" %% "s3"      % "0.1.0-11-ge66487e-SNAPSHOT",
+  "ohnosequences" %% "forests" % "0.1.0-2-gdbfdd11-SNAPSHOT"
```

Comment:
Should we release new versions of these packages?

Reply:
Yep, we should